src/gui_common/charts/line/LineChart.cs

```diff
@@ -693,6 +696,19 @@ private void UpdateLineColliders(string datasetName, DataPoint firstPoint, DataP
         mouseCollider.Visible = dataSets[datasetName].Draw;
     }
 
+    /// <summary>
+    ///   Draws a text on the chart clarifying that there's no data to show to the user.
+    /// </summary>
+    private void DrawNoDataText()
+    {
+        var font = GetFont("lato_small", "Label");
+
+        var position = new Vector2((drawArea.RectSize.x - font.GetStringSize(
+            TranslationServer.Translate("NO_DATA_TO_SHOW")).x) / 2, drawArea.RectSize.y / 2);
```

Comment:
I find the computations pretty hard to read as such with formatting

Reply:
I think this is due to the translation server call as this calculation is already at its simplest form, will put the translate call to a local variable.

Reply:
Yeah, I also find it much more readable now that computations for x aren't split on two lines, with one further sharing computations for y.